crates/state/src/lib.rs

```diff
@@ -48,6 +48,11 @@ pub type Result<T> = std::result::Result<T, Error>;
 /// it has 2 blocks delay on validator set update.
 pub const EPOCH_SWITCH_BLOCKS_DELAY: u32 = 2;
 
+/// Returns true if the given key is one whose data is not merklized
+// pub fn is_key_not_merklized(key: &Key) -> bool {
+//     is_masp_key(key) || is_ibc_counter_key(key)
+// }
+
```

Comment:
just a note here to not forget to remove this and the test below if we don't use it